src/tests/system/tests/test_passkey.py

```diff
@@ -740,3 +740,151 @@ def test_passkey__su_with_12_mappings(
         )
         in output
     ), "Get the console message about TGT"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.builtwith(client="passkey", ipa="passkey")
+@pytest.mark.ticket(gh=6931)
+def test_passkey__su_no_pin_set(
+    client: Client, ipa: IPA, moduledatadir: str, testdatadir: str, umockdev_ipaotpd_update
+):
+    """
+    :title: Check authentication of user with IPA server when no pin set for the Passkey
+    :setup:
+        1. Add a user with --user-auth-type=passkey in the IPA server
+        2. Modify Passkey configuration to set require user verification during authentication to false
+        3. Setup SSSD client with FIDO and umockdev, start SSSD service
+    :steps:
+         1. Check authentication of the user when no pin set for the Passkey
+         2. Check the TGT of user
+    :expectedresults:
+        1. User authenticates successfully
+        2. Get TGT after authentication of user
+    :customerscenario: False
+    """
+    with open(f"{testdatadir}/passkey-mapping.ipa") as f:
+        ipa.user("user1").add(user_auth_type="passkey").passkey_add(f.read().strip())
+
+    ipa.host.conn.run("ipa passkeyconfig-mod --require-user-verification=False", raise_on_error=False)
+    client.sssd.start(service_user="root")
+
+    rc, _, output, _ = client.auth.su.passkey_with_output(
+        username="user1",
+        device=f"{moduledatadir}/umockdev.device",
+        ioctl=f"{moduledatadir}/umockdev.ioctl",
+        script=f"{testdatadir}/umockdev.script.ipa",
+        command="klist",
+        auth_method="PasskeyAuthenticationUseCases.PASSKEY_NO_PIN_NO_PROMPTS",
+    )
+
+    ipa.host.conn.run("ipa passkeyconfig-mod --require-user-verification=True", raise_on_error=False)
+
+    assert rc == 0, "Authentication failed"
+    assert "Ticket cache" in output, "Failed to get the TGT"
+    assert not (
+        (
+            "No Kerberos TGT granted as the server does not support this method. "
+            "Your single-sign on(SSO) experience will be affected"
+        )
+        in output
+    ), "Get the console message about TGT"
```

Comment:
It looks like this block of code is reused often here, perhaps put it into its own utility method like `assert_passkey_auth_success()`

```
    assert rc == 0, "Authentication failed"
    assert "Ticket cache" in output, "Failed to get the TGT"
    assert not (
        (
            "No Kerberos TGT granted as the server does not support this method. "
            "Your single-sign on(SSO) experience will be affected"
        )
        in output
    ), "Get the console message about TGT"
```

Reply:
We can do that but if I understand correctly then every time we need to check what we define in method and may be not easy readable for others who did not know about Passkey, but we can do that.

Reply:
I'll leave it up to you to decide, I don't insist.